### DISCUSSION TITLE
Why was RWKV package removed?

### DISCUSSION THREAD

**Discussion by ye7iaserag** [oobabooga/text-generation-webui#422] [General]
Asking about this commit https://github.com/oobabooga/text-generation-webui/commit/86b99006d9c2aea21aa8d71413560246737e3c57

**Comment by Daviljoe193** [comment]
And now it's back? Uh... yay? https://github.com/oobabooga/text-generation-webui/commit/7073e960933e8c2aad1881d57da1b42303cca528
